doc/qbittorrent.1.md

```diff
@@ -1,36 +1,236 @@
+
+`--relative-fastresume`
+
+:    Hack into libtorrent _.fastresume_ files and make file paths relative to the **`--profile`** directory. "If you have to ask, you don't need this."
+
+`--webui-port=`_NUMBER_
+
+:    Specify the port _NUMBER_ the WebUI should listen on. The default if not specified is 8080.
+
+## Options to control added torrents
+
+These options only apply to the added torrents passed in \[_URI_...\]. If no torrent is specified, these options are ignored.
+
+`--add-paused=`_TOGGLE_
+
+:    _TOGGLE_ is one of **true** or **false**. If **true**, the torrent(s) will not be automatically started. Defaults to **false**. Note that if a torrent's files exist in the destination save path, a recheck will start, unless **`--skip-hash-check`** is used.
+
+`--category=`_NAME_
+
+:    Assign the torrent(s) to the category _NAME_. If this category does not exist, it will be automatically created.
+
+`--first-and-last`
+
+:    Download first and last pieces first.
+
+`--save-path=`_PATH_
+:    Save the torrent(s) to the specified _PATH_. A relative _PATH_ may be used, in which case it will be relative to the current working directory that qBittorrent is started from. Note that **`~`** will not get expanded, and will be interpreted as a folder literally named **`~`**.
+
+`--sequential`
+:    Download pieces in sequential order.
+
+`--skip-dialog=`_TOGGLE_
+
+:    _TOGGLE_ is one of **true** or **false**. If **true**, the "Add New Torrent" dialog will not be shown when adding torrents. Defaults to **false**.
+
+`--skip-hash-check`
+
+:    By default, if a torrent's files exist in the destination save path, a recheck is initiated. This option prevents that from happening, which is not recommended in general. Torrents may still be force-rechecked after they are added, as usual. If the torrent's files do not exist in the destination save path, this option has no effect.
+
+# ENVIRONMENT
+
+_OPTIONS_ may be supplied via environment variables with equivalent names.
+
+For an option named **parameter-name**, the corresponding environment variable name is **QBT\_PARAMETER\_NAME**. In other words, the parameter name is converted to upper-case, any hyphens (**-**) are replaced by underscores (**\_**), and **QBT\_** is prepended to it.
+
+To pass flag values, set the variable to **1** or **TRUE**. For example, **`--add-paused=true`** becomes **QBT\_ADD\_PAUSED=1**
+
+# FILES
+
+## Standard configuration files directory hierarchy
 
-**`--no-splash`** Disables splash screen on startup.
+Currently, qBittorrent complies somewhat with the Freedesktop XDG Base Directory specification. The configuration files are stored in the default directories of the specification:
 
-**`--webui-port=x`** Changes Web UI port to x (default: 8080).
+1. Configuration files are stored in _~/.config/qBittorrent_
+2. Data files are stored in _~/.local/share/data/qBittorrent_ (in the future this will be changed to just _~/.local/share/qBittorrent_, in a backwards-compatible way)
```

Comment:
ughh.. why give out promise when there is no ETA?

Reply:
Yeah, probably best to just update the man page when it is fixed. Fixed.